refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,758 @@
+- **Complete** : They allow to express the totality of possible needs in the context of a user wanting to [regulate their privacy/connectedness](https://github.com/blindnet-io/product-management/blob/dogma/refs/notion-of-privacy/notion-of-privacy.md), as well as the totality of requests defined by the [supported legilsation](#supported-legilsation).
+
+### Extending the vocabulary
+
+Systems MAY specify the vocabulary used to express data being exchanged. The value indicating this version of this vocabulary is `priv.1.0`.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `vocab` | 0-1 | The name of the vocabulary used to describe data. In absence of indication `priv.1.0` is assumed. |
+
+This vocabulary can be extended by defining a new identifier for the new vocabulary. New vocabularies MAY include other vocabularies as long as their sets of concepts, properties and terms are disjoint.  
+
+In addition, the Systems implementing PRIV can simply define their own subcategories of terms (not properties not concepts) defined by PRIV. This might be useful for their own internal operation, however the interoperability with other system can only be expected at the level of the particular, shared, version of the vocabulary.
+
+## Examples
+
+A [JSON schema of the PRIV](./priv.schema.json) is provided for convenience. To illustrate a request, let us take the example of a data subject wanting to know if a an organisation (and any of its partners if applicable) have any data on them, and if so, have their contact data deleted.
+
+This request can be represented with the following json data:
+```
+{
+  "$schema":"https://blindnet.io/schemas/priv.schema.json",
+  "request-id": "8f9066c6-1c6c-42a0-9993-e88c98d0e84d",
+  "date": "2022-06-02T14:40:39+0000",
+  "data-subject":[{
+    "dsid-schema": "email-sha-256",
+    "dsid":"7cac89a56bbf998c996f33e0b2d3bad578e05f3af8d64793c0bcac46b8c260dc"
+  }],
+  "demands":[{
+    "demand-id":"496294eb-5293-47dd-aaf8-494a0cb09134",
+    "action":"TRANSPARENCY.KNOWN"
+  },{
+    "demand-id":"86bbb28a-eee6-45e6-81d6-7101de32374b",
+    "action":"DELETE",
+    "restrictions":[{
+      "data-categories":["CONTACT"]
+    }]
+  }],
+  "target":"PARTNERS"
+
+}
+
+```
+In [here](./examples.md) we provide an overview of various Privacy Requests that a Data Subject might want to formulate according to [supported legislation](#supported-legislation) and we give indications on how each of those can be modelled with the Privacy Request Interchange Vocabulary.
+
+## Design Implications for Systems Implementing PRIV
+
+### Remembering Transfers
+
+When data about Data Subjects is transmitted from one system to another, in order to be able to process the [Targets property](#targets), and in order to reply to `TRANSPARENCY.WHO` and `TRANSPARENCY.PROVENANCE` demands, Systems MUST keep track of:
+- System of destination/origin and addresses where their APIs can be reached
```

Comment:
What if another system can't be reached or deliberately refuses requests?

Reply:
We need a protocol for that (system adressability, discovery, and exchange of Privacy Requests)